### ISSUE TITLE
How to fuse two different architecture softmax score?

### ISSUE THREAD

**Issue by Zolza09** [wushidonguc/Two-stream-action-recognition-keras#5]
Hi, 
I am modifying your code and using another dataset. 
1.  Why temporal and spatial streams are using different architecture? temporal using AlexNet architecture and spatial using Inception architecture. 
2.  Can we fuse two different architecture? 
3.  After training temporal and spatial all of them accuracy not increasing and even not saved the weights in output?
